tartufo/scanner.py

```diff
@@ -749,16 +749,15 @@ class FolderScanner(ScannerBase):
     target: str
 
     def __init__(
-        self,
-        global_options: types.GlobalOptions,
-        target: str,
+        self, global_options: types.GlobalOptions, target: str, recurse: bool
```

Comment:
If we default this to true here, we can avoid changing a bunch of tests
```suggestion
        self, global_options: types.GlobalOptions, target: str, recurse: bool = True
```

But then again, we don't do this for options like `include-submodules` so I'll defer to other about whether we should do this or not.

Reply:
I will skip this change to maintain consistency for now unless we want to have this change.

Reply:
@jwilhelm-godaddy  @rbailey-godaddy What are your thoughts about this?

Reply:
I think it's fine to leave this as-is. tartufo has been built as a library first, with a CLI that just happens to use that library. With that in mind, the library tries to operate with as few assumptions as possible. So if we can avoid an assumption here and remain more explicit, I would say that's preferable.